src/Extensions/TimedScopes/Internal/TimedScopeEventSender.cs

```diff
@@ -20,31 +23,46 @@ public TimedScopeEventSender(TimedScopeEventSource eventSource, IHostEnvironment
 		}
 
 
-		public void LogTimedScopeEndEvent(TimedScope scope)
+		public void LogActivityStop(Activity activity)
 		{
 			if (!m_eventSource.IsEnabled())
 			{
 				return;
 			}
 
 			string serviceName = m_serviceName;
-			string subtype = scope.SubType;
-			string metadata = scope.Metadata;
-			TimedScopeResult result = scope.Result;
-			Activity activity = scope.Activity;
 			string name = activity.OperationName;
 			string correlationId = activity.Id;
 			double durationMs = activity.Duration.TotalMilliseconds;
 			string userHash = activity.GetUserHash(); //TODO: We need add middleware that will set userhash in compliant way and IsTransaction GitHub Issue #166
 			bool isTransaction = activity.IsTransaction();
 
+
+			string subtype = NullPlaceholder;
```

Comment:
Is there a reason to set these to "null" by default instead of ""?

Reply:
I've thought it's some kind of ETW requirement, I've actually wanted to ask you about it :)

Reply:
I have no idea. If it was done like this before then leave it like that.

Reply:
I'll leave it like this, for now, it's not allocating almost anything, but I'll try to investigate it in future

Reply:
From short googling ti looks like event source might fail if you pass null instead of a sting, but I'll create a test a bit later

Reply:
I was thinking "", so string.Empty, not null.